### PR TITLE
feat: Add option to render pagination content inside specified element.

### PR DESCRIPTION
**Pull request by T-Wizard**
For use in Analytics view where pagination is combined with extra content

Example Image:
table

### CODE REVIEW

**Comment by LiKang6688** [comment]
Can you post a screenshot here, please?

**Comment by streamside** [comment]
I suggest to use `footer` as name for the option. Over time I suppose the content below the table may change and have other things than only pagination features.

**Comment by LiKang6688** [comment]
Hi @T-Wizard, I agree with @streamside. Could you rename it?

**Comment by T-Wizard** [comment]
> Hi @T-Wizard, I agree with @streamside. Could you rename it?

do you want the name to be just `footer`?
or should it be `footerContainer` to indicate that the footer will be placerd inside it. (and not an option to replace the footer content)

**Comment by LiKang6688** [comment]
> > Hi @T-Wizard, I agree with @streamside. Could you rename it?
> 
> do you want the name to be just `footer`? or should it be `footerContainer` to indicate that the footer will be placerd inside it. (and not an option to replace the footer content)

footerContainer looks good to me.

**Review by LiKang6688 — APPROVED** [review]
LGTM

**Comment by T-Wizard** [comment]
@haxxmaxx @niekvanstaveren any comment? (Or should i merge it)

File: src/table/components/TableWrapper.jsx [hunk 1]
```diff
@@ -234,6 +249,7 @@ TableWrapper.propTypes = {
   selectionsAPI: PropTypes.object.isRequired,
   keyboard: PropTypes.object.isRequired,
   rect: PropTypes.object.isRequired,
+  footerContainer: PropTypes.object.isRequired,
```
Comment (niekvanstaveren) [inline src/table/components/TableWrapper.jsx:252]: Should this be required?

**Review by niekvanstaveren — APPROVED** [review]
Looks good. I don't think the footerContainer should be required though

File: src/table/components/TableWrapper.jsx (hunk 1 above)
Comment (T-Wizard) [reply src/table/components/TableWrapper.jsx:252]: I will change it to not required.

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**